classes/base.lua

```diff
@@ -101,10 +102,17 @@ function class:declareOptions ()
     end
     return self._name
   end)
+  self:declareOption("landscape", function(_, landscape)
+    if landscape then
+      self.landscape = landscape
+    end
+    SILE.documentState.landscape = self.landscape
```

Comment:
```suggestion
    if landscape then
      self.landscape = SU.boolean(landscape, false)
      SILE.documentState.landscape = self.landscape
    end
```

Lets again use our boolean helper here. More importantly this option handler is both a setter and a getter, so the "set" action being done touching the document state should only happen in the if condition that runs when this is used as a setter, i.e. when called with a value.

Reply:
Right, all these suggestions were committed.